apps/projects/app/components/Content/IssueDetail/BountyCard.js

```diff
@@ -0,0 +1,229 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {
+  Box,
+  Button,
+  GU,
+  IconAddUser,
+  IconClock,
+  IconFile,
+  IconGraph2,
+  IdentityBadge,
+  Link,
+  Text,
+  useTheme,
+} from '@aragon/ui'
+import { issueShape } from '../../../utils/shapes.js'
+import { usePanelManagement } from '../../Panel'
+import { useAragonApi } from '../../../api-react'
+import { formatDistance } from 'date-fns'
+import { IconDoubleCheck, IconUserCheck } from '../../../assets'
+
+const ActionButton = ({ panel, caption, issue }) => (
+  <EventButton
+    mode="normal"
+    wide
+    onClick={() => panel(issue)}
+  >
+    <Text size="large">
+      {caption}
+    </Text>
+  </EventButton>
+)
+ActionButton.propTypes = {
+  panel: PropTypes.func.isRequired,
+  caption: PropTypes.string.isRequired,
+  issue: PropTypes.oneOfType([
+    issueShape,
+    PropTypes.arrayOf(issueShape),
+  ]).isRequired,
+}
+
+const address = address => (
+  <div css={`margin-left: ${GU}px`}>
+    <IdentityBadge entity={address} />
+  </div>
+)
+
+const DeadlineDistance = ({ date }) =>
+  formatDistance(new Date(date), new Date(), { addSuffix: true })
+
+const plural = number => number > 1 ? 's' : ''
```

Comment:
Note that my second example does not keep the interface of packages like `pluralize`, so we may not want it. But it makes our code nicer now. I'll leave that decision up to you.

Reply:
Depending on what's inside `pluralize` we can change 3 places in the code like this:

`{issue.requestsData.length + ' application' + plural(issue.requestsData.length)}`
vs
`{issue.requestsData.length + pluralize('application', issue.requestsData.length)}`
vs
`{pluralize('application', issue.requestsData.length)}`

If we get to the point of needing to pluralize words more interesting than `application`, it will be a conscious decision and changing how we call it will cost... insignificant effort. On the other hand considering what we are getting from the last version right now - it makes sense to use that one...